src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -665,6 +663,40 @@ static int add_partition_root_rw(DissectedPartition *p) {
         return 0;
 }
 
+static int add_partition_root_growfs(DissectedPartition *p) {
```

Comment:
This can return void?

Reply:
yeah, it could, but i'd keep this similar to add_partition_root_rw().

dunno, it's a bit of bikeshedding thing, no? compiler would remove it anyway...

i kinda like the symmetrics of code hre, i.e. that add_partition_root_flags() dispatches to a separate function for each flag. dunno.

Reply:
Then I think it's better if we honor all flags, but collect them using RET_GATHER 🤔

Reply:
addressed